crates/cli/src/lib.rs

```diff
@@ -0,0 +1,50 @@
+use std::fs;
```

Comment:
please docstrings on modules types and functions they really help me more than you think.

Reply:
Oh, thank you for the reminder! I will add lots of comments.

Reply:
Done.